src/payment/strategies/offsite-payment-strategy.ts

```diff
@@ -42,4 +42,11 @@ export default class OffsitePaymentStrategy extends PaymentStrategy {
 
         return super.finalize();
     }
+
+    shouldSubmitFullPayload(payment: any): boolean {
+        return payment && (
```

Comment:
@davidchin we're still checking the `methodId` here because I haven't yet figured out a way to populate `gatewayId` in this method:

![screen shot 2018-09-21 at 3 13 32 pm](https://user-images.githubusercontent.com/2459655/45905097-c9ab2680-bdb4-11e8-9ebf-1209f8447d37.png)

However without this check the initialize request fails entirely, like so:

![screen shot 2018-09-21 at 3 07 34 pm](https://user-images.githubusercontent.com/2459655/45905114-df205080-bdb4-11e8-9039-7b6f411a0b5f.png)